scripts/install.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
This file is mostly an updated version of `build.sh` focused on installation, with actual builds driven by `make`.